nil pointer panic when trying to distrust a nonexistent keypair

/kind bug

**1. What `kops` version are you running? The command `kops version`, will display
 this information.**
1.24.0

**2. What Kubernetes version are you running? `kubectl version` will print the
 version if a cluster is running or provide the Kubernetes version specified as
 a `kops` flag.**
n/a

**3. What cloud provider are you using?**
AWS

**4. What commands did you run?  What is the simplest way to reproduce this issue?**
```bash
kops distrust keypair this-is-not-a-keypair 12345678910
```

**5. What happened after the commands executed?**

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x592e160]

goroutine 1 [running]:
main.distrustKeypair({0x73b2fc0, 0xc000130008}, {0x7fffa2f032fe, 0xf}, {0xc0000c47d0, 0x1, 0x1}, {0x74f7fd8, 0xc000ceba70})
	k8s.io/kops/cmd/kops/distrust_keypair.go:168 +0x3a0
main.RunDistrustKeypair({0x74b20c0, 0xc00012e008}, 0x0, {0x73b2fc0, 0xc000130008}, 0xc000bda040)
	k8s.io/kops/cmd/kops/distrust_keypair.go:128 +0x11a
main.NewCmdDistrustKeypair.func3(0xc000a30280, {0xc0000c47c0, 0x2, 0x2})
	k8s.io/kops/cmd/kops/distrust_keypair.go:104 +0x37
k8s.io/kops/vendor/github.com/spf13/cobra.(*Command).execute(0xc000a30280, {0xc0000c4780, 0x2, 0x2})
	k8s.io/kops/vendor/github.com/spf13/cobra/command.go:872 +0x624
k8s.io/kops/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x9e7ca80)
	k8s.io/kops/vendor/github.com/spf13/cobra/command.go:990 +0x3bc
k8s.io/kops/vendor/github.com/spf13/cobra.(*Command).Execute(...)
	k8s.io/kops/vendor/github.com/spf13/cobra/command.go:918
main.Execute()
	k8s.io/kops/cmd/kops/root.go:95 +0x5c
main.main()
	k8s.io/kops/cmd/kops/main.go:20 +0x17
```

**6. What did you expect to happen?**
Friendly error message and graceful exit.

**7. Please provide your cluster manifest. Execute
  `kops get --name my.example.com -o yaml` to display your cluster manifest.
  You may want to remove your cluster name and other sensitive information.**
n/a

**8. Please run the commands with most verbose logging by adding the `-v 10` flag.
  Paste the logs into this report, or in a gist and provide the gist link here.**

```
I0727 19:00:54.932079 1870317 s3fs.go:329] Reading file "s3://my-state-store/my-cluster.local/private/this-is-not-a-keypair/keyset.yaml"
```

Then the same as above.

**9. Anything else do we need to know?**

PR forthcoming :smile:

--- a/cmd/kops/distrust_keypair.go
+++ b/cmd/kops/distrust_keypair.go
@@ -148,6 +148,8 @@ func distrustKeypair(out io.Writer, name string, keypairIDs []string, keyStore f
 	keyset, err := keyStore.FindKeyset(name)
 	if err != nil {
 		return err
+	} else if keyset == nil {
+		return fmt.Errorf("keyset not found")
 	}
 
 	if len(keypairIDs) == 0 {

